partcad/tests/unit/test_sketch.py

```
@@ -0,0 +1,35 @@
+def test_sketch_get_dxf():
+    """Load a DXF sketch from a project by the part name"""
+    ctx = pc.Context("examples/produce_sketch_dxf")
+    repo1 = ctx.get_project(".")
+    assert repo1 is not None
+    sketch = repo1.get_sketch("dxf_01")
+    assert sketch is not None
+    wrapped = asyncio.run(sketch.get_wrapped())
+    assert wrapped is not None
+
+
+def test_sketch_get_svg():
```

Ruff (0.8.2)

Missing return type annotation for public function `test_sketch_get_svg`

Add return type annotation: `None`

(ANN201)